index.html

```diff
@@ -1,3 +1,134 @@
+                    <li>Canopy</li>
+                    <li>Black end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+                </p>
+                <ul class="grid1">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button>See Project</button><br>
+
+            </div>
+            <br>
+        </div>
+
+        <div class="main-section">
+            <div class="main-image">
+                <img src="img/markus-spiske-ZSZ6wzNU12Q-unsplash.jpg" alt="main image">
+            </div>
+            <div class="text-section">
+                <h1>Tonic</h1>
+                <ul class="navi">
+                    <li>Canopy</li>
+                    <li>Black end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+                </p>
+                <ul class="grid1">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button>See Project</button>
+
+            </div>
+        </div>
+        <div class="about-section">
+            <h1>About Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p>LETS CONNECT</p>
+            <ul class="footer-section">
+                <li><a href="#"><img src="img/github@2x.png" alt="github.com"></a></li>
+                <li><a href="#"><img src="img/Linkedinicon@2x.png" alt="Linkedin"></a></li>
+                <li><a href="#"><img src="img/twitter@2x.png" alt="twitter"></a></li>
+                <li><a href="#"><img src="img/medium@2x.png" alt="Medium"></a></li>
+            </ul>
+            <button>Get my resume</button>
+
+            <h2>Languages</h2>
+                <ul id="lang">
+                    <li>Javascript</li>
+                    <li>Html</li>
+                    <li>CSS</li>
+                </ul>
+            <h2>Frameworks</h2>
+            <h2>Skills</h2>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be a list, each of them in a li
The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul, and each of them a li